### PR TITLE
Update pet name when changing launch template id

### PR DESCRIPTION
**Pull request by woz5999**
## what
* Update pet name when changing launch template id

## why
* If you change the template id, tf/aws will try to create a new node group; however, because we're using create before destroy, it errors out due to "NodeGroup already exists with name food and cluster name bar"
* generating a new pet name will restore the correct behavior
* note that this was the behavior prior to https://github.com/cloudposse/terraform-aws-eks-node-group/pull/84

### CODE REVIEW

File: context.tf [hunk 1]
```diff
@@ -20,6 +20,7 @@
 # will be null, and `module.this.delimiter` will be `-` (hyphen).
 #
 
+
```
Comment (aknysh) [inline context.tf:23]: why this file has been changed?
It should be taken from https://github.com/cloudposse/terraform-null-label/blob/master/exports/context.tf

File: examples/complete/context.tf [hunk 2]
```diff
@@ -20,6 +20,7 @@
 # will be null, and `module.this.delimiter` will be `-` (hyphen).
 #
 
+
```
Comment (aknysh) [inline examples/complete/context.tf:23]: why this file has been changed?
It should be taken from https://github.com/cloudposse/terraform-null-label/blob/master/exports/context.tf

Comment (woz5999) [reply examples/complete/context.tf:23]: ah sorry. i see a pushed of pollution from when i updated my fork. i'll fix everything up nice. i'll set and unset draft status accordingly.

**Pull request closed without merging**